pkg/resource/aws/deserializer/sns_topic_policy_deserializer.go

```diff
@@ -0,0 +1,39 @@
+package deserializer
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/gocty"
+)
+
+type SNSTopicPolicyDeserializer struct {
+}
+
+func NewSNSTopicPolicyDeserializer() *SNSTopicPolicyDeserializer {
+	return &SNSTopicPolicyDeserializer{}
+}
+
+func (s *SNSTopicPolicyDeserializer) HandledType() resource.ResourceType {
+	return aws.AwsSnsTopicPolicyResourceType
+}
+
+func (s SNSTopicPolicyDeserializer) Deserialize(topicList []cty.Value) ([]resource.Resource, error) {
+	policies := make([]resource.Resource, 0)
+
+	for _, value := range topicList {
+		policy, err := decodeSNSTopicPolicy(value)
```

Comment:
We should shadow the variable here from now on to be consistent with what @elie has done on his last PR. 

```go
for _, value := range topicList {
    value := value
    policy, err := decodeSNSTopicPolicy(value)
```

Reply:
done

Reply:
it does not seem to trigger linter error, I'm wondering if this is required there

Reply:
I edited it anyway